README.md

```diff
@@ -21,14 +21,61 @@ The product is anticipated to include:
 3. Enhancements to the public-facing [BC Mine Information website](http://mines.nrs.gov.bc.ca/) ([github repository](https://github.com/bcgov/mem-mmti-public)) to broaden the available data and create new or improved functionality to increase usability
 4. Enhancements to the [BC Mine Information website's administrative console](https://mines.empr.gov.bc.ca/) ([github repository](https://github.com/bcgov/mem-admin)) , including work required to connect with other systems (e.g., the Natural Resource Inspection System, aka NRIS)
 
+## Services
+
+// suggestion: simple description of each, perhaps organize into categories. Sub-files can either by easier to discover at top level README or more concise by linking within
+[Common: empty file, suggest delete or add content](services/common/README.md)
+[Core Web (Ministry Frontend)](services/core-web/README.md)
+[Minespace Web (Proponent Frontend)](services/minespace-web/README.md)
+[Core API (Shared Backend)](services/core-api/README.md)
+
+- [Core API JWT](services/core-api/app/flask_jwt_oidc_local/README.md)
+  [Database](services/database/README.md)
+- [Database Migrations](migrations/README.md)
+  [Document Manager](services/document-manager/backend/README.md)
+  - [Document Manager Migrations: suggest deleting empty file](services/document-manager/backend/migrations/README.md)
+    [Fider](services/fider/README.md)
+    [Filesystem Provider](services/filesystem-provider/ej2-amazon-s3-aspcore-file-provider/README.md)
+
+## Integrations
+
+[NRIS](services/nris-api/backend/README.md)
+
+## Operations
+
+[Azure](operations/azure/README.md)
+
+- [Azure Setup](operations/azure/setup/README.md)
+
+## Tests
+
+// ISSUE WITH TESTING: tests running in sub-directories frequently use wrong node version (I think) and fail to run
+// re-examine this
+[Testing Strategy](docs/testing/test_strategy.md)
+// Do we do this?
+[Functional Tests](tests/functional-tests/README.md)
+// Or this?
+[Load Testing](tests/load-testing/README.md)
+
 ## How to Contribute
 
-Please read the [How to Contribute guide](CONTRIBUTING.md).
+Please read the [How to Contribute guide](CONTRIBUTING.md) and the [Code of Conduct](CODE_OF_CONDUCT.md).
 
 ## How to Develop
 
 Please read the [How to develop](USAGE.md).
+// There is overlapping/related content in the following files: suggest combining all 3 and adding updates
+[Getting Started](docs/devops/getting_started.md)
+[Monorepo setup](docs/frontend/monorepo.md)
+
+### OpenShift Deployment
+
+[OpenShift Debugging Guide](docs/openshift/debugging_guide.md)
+[OpenShift Caveats](docs/openshift/Openshift%20Caveats.md)
+[Terraform](terraform/README.md)
+[OpenShift Database: suggest deletion](docs/openshift/database.md)
+[OpenShift PG upgrade: suggest deletion](docs/openshift/PG_9_to_13_upgrade.md)
```

Comment:
@vyasworks any value in keeping the OpenShift Database and OpenShift PG update readme files?